core/chains/solana/monitor/balance.go

```diff
@@ -6,9 +6,9 @@ import (
 
 	"github.com/gagliardetto/solana-go"
 
+	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
 	solanaClient "github.com/smartcontractkit/chainlink-solana/pkg/solana/client"
 
-	"github.com/smartcontractkit/chainlink/v2/core/logger"
```

Comment:
> The bad news is that we need to switch over plugin-side code to use the relay Logger instead of core, which has a lot of noisy side-effects, but that needed to happen eventually anyways.

The plugin side code. We were using the core logger as a convenience, but plugins shouldn't be importing core and won't be able to when we move them to the chain family repos.